download.html

```diff
@@ -264,6 +264,24 @@ <h3>OpenSUSE</h3>
                   <h3>CentOS</h3>
                   <p> <a href="http://geekery.altervista.org/dokuwiki/doku.php">Contributor Packages</a> </p>
                </div>
+               <!-- Alpine Linux -->
+               <div class="col-md-3 themed-grid-col pb-3"> <img src="/assets/images/distro_logos/alpinelinux_logo.png"
```

Comment:
Is there a SVG version? Seems most other distros use SVG here. Not critical and could merge as is, but thought I'd ask.

Reply:
Yes. I found the SVG and replaced the PNG.